src/lib.rs

```diff
@@ -13,12 +13,47 @@ extern "C" {
     fn ethereum_getBlockTimestamp() -> u64;
     fn ethereum_getTxGasPrice(valueOffset: *const u32);
     fn ethereum_getTxOrigin(resultOffset: *const u32);
-    fn ethereum_log(dataOffset: *const u32, length: u32, numberOfTopics: u32, topic1: *const u32, topic2: *const u32, topic3: *const u32, topic4: *const u32);
-    fn ethereum_call(gas: u64, addressOffset: *const u32, valueOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_callCode(gas: u64, addressOffset: *const u32, valueOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_callDelegate(gas: u64, addressOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_callStatic(gas: u64, addressOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_create(valueOffset: *const u32, dataOffset: *const u32, dataLength: u32, resultOffset: *const u32) -> u32;
+    fn ethereum_log(
+        dataOffset: *const u32,
+        length: u32,
+        numberOfTopics: u32,
+        topic1: *const u32,
+        topic2: *const u32,
+        topic3: *const u32,
+        topic4: *const u32,
```

Comment:
Can you change the rustfmt configuration so that trailing commas are not added?

Reply:
nightly feature, unfortunately. Would rather stick to stable